components/javascript/src/main/java/org/datacleaner/beans/script/JavaScriptTransformer.java

```diff
@@ -52,9 +52,7 @@
 	private static final Logger logger = LoggerFactory
 			.getLogger(JavaScriptTransformer.class);
 
-	public static enum ReturnType {
-		STRING, NUMBER, BOOLEAN;
-	}
+	public enum ReturnType {STRING, NUMBER, BOOLEAN}
```

Comment:
Hmmm, first point probably true, but the last point only really matters when we actually DO implement behaviour, it shouldn't affect how a traditional ("stupid") enum is represented... Actually it could be a nice way to tell them apart.